client/my-sites/migrate/section-migrate.jsx

```diff
@@ -62,9 +69,13 @@ class SectionMigrate extends Component {
 	resetMigration = () => {
 		const { targetSiteId, targetSiteSlug } = this.props;
 
-		wpcom.resetMigration( targetSiteId ).then( () => {
+		wpcom.resetMigration( targetSiteId ).finally( () => {
 			page( `/migrate/${ targetSiteSlug }` );
-			this.updateFromAPI();
+			/**
+			 * Note this migrationStatus is local, thus the setState vs setMigrationState.
+			 * Call to updateFromAPI will update non-local state.
```

Comment:
This line's a bit ambiguous – won't the response to `updateFromAPI` also set a local state?

Reply:
Right.. it'll update both local and non-local state. I'll update the comment to be more clear.